From 2db7293cf1ca6af2f3a8a43bb4772c0e3a69846f Mon Sep 17 00:00:00 2001
From: Abhay Ruparel <abhayruparel2000@gmail.com>
Date: Tue, 26 Jun 2018 22:42:36 +0530
Subject: [PATCH 2/6] native: drop hwui property on O-MR1

 * In Android 8.1 and later, only the ro.zygote.disable_gl_preload
   property still applies. All other properties have been removed.
   Reference: source.android.com/devices/graphics/renderer

Change-Id: Id791029a85bd2ae3ed1c8f38257ea123ac25e61c
Signed-off-by: Abhay Ruparel <abhayruparel2000@gmail.com>
---
 build/phone-xxhdpi-2048-hwui-memory.mk  | 30 -------------------------
 build/phone-xxhdpi-3072-hwui-memory.mk  | 30 -------------------------
 build/phone-xxxhdpi-3072-hwui-memory.mk | 30 -------------------------
 3 files changed, 90 deletions(-)
 delete mode 100644 build/phone-xxhdpi-2048-hwui-memory.mk
 delete mode 100644 build/phone-xxhdpi-3072-hwui-memory.mk
 delete mode 100644 build/phone-xxxhdpi-3072-hwui-memory.mk

diff --git a/build/phone-xxhdpi-2048-hwui-memory.mk b/build/phone-xxhdpi-2048-hwui-memory.mk
deleted file mode 100644
index b4226bc4a..000000000
--- a/build/phone-xxhdpi-2048-hwui-memory.mk
+++ /dev/null
@@ -1,30 +0,0 @@
-#
-# Copyright (C) 2013 The CyanogenMod Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-# Provides overrides to configure the HWUI memory limits
-
-PRODUCT_PROPERTY_OVERRIDES += \
-    ro.hwui.texture_cache_size=72 \
-    ro.hwui.layer_cache_size=48 \
-    ro.hwui.r_buffer_cache_size=8 \
-    ro.hwui.path_cache_size=32 \
-    ro.hwui.gradient_cache_size=1 \
-    ro.hwui.drop_shadow_cache_size=6 \
-    ro.hwui.texture_cache_flushrate=0.4 \
-    ro.hwui.text_small_cache_width=1024 \
-    ro.hwui.text_small_cache_height=1024 \
-    ro.hwui.text_large_cache_width=2048 \
-    ro.hwui.text_large_cache_height=1024
diff --git a/build/phone-xxhdpi-3072-hwui-memory.mk b/build/phone-xxhdpi-3072-hwui-memory.mk
deleted file mode 100644
index 217bfb845..000000000
--- a/build/phone-xxhdpi-3072-hwui-memory.mk
+++ /dev/null
@@ -1,30 +0,0 @@
-#
-# Copyright (C) 2016 The CyanogenMod Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-# Provides overrides to configure the HWUI memory limits
-
-PRODUCT_PROPERTY_OVERRIDES += \
-    ro.hwui.texture_cache_size=72 \
-    ro.hwui.layer_cache_size=48 \
-    ro.hwui.path_cache_size=32 \
-    ro.hwui.gradient_cache_size=1 \
-    ro.hwui.drop_shadow_cache_size=6 \
-    ro.hwui.r_buffer_cache_size=8 \
-    ro.hwui.texture_cache_flushrate=0.4 \
-    ro.hwui.text_small_cache_width=1024 \
-    ro.hwui.text_small_cache_height=1024 \
-    ro.hwui.text_large_cache_width=2048 \
-    ro.hwui.text_large_cache_height=1024
diff --git a/build/phone-xxxhdpi-3072-hwui-memory.mk b/build/phone-xxxhdpi-3072-hwui-memory.mk
deleted file mode 100644
index 8123fa00b..000000000
--- a/build/phone-xxxhdpi-3072-hwui-memory.mk
+++ /dev/null
@@ -1,30 +0,0 @@
-#
-# Copyright (C) 2015 The CyanogenMod Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-# Provides overrides to configure the HWUI memory limits
-
-PRODUCT_PROPERTY_OVERRIDES += \
-    ro.hwui.texture_cache_size=88 \
-    ro.hwui.layer_cache_size=58 \
-    ro.hwui.path_cache_size=32 \
-    ro.hwui.shape_cache_size=4 \
-    ro.hwui.gradient_cache_size=2 \
-    ro.hwui.drop_shadow_cache_size=8 \
-    ro.hwui.r_buffer_cache_size=8 \
-    ro.hwui.text_small_cache_width=2048 \
-    ro.hwui.text_small_cache_height=2048 \
-    ro.hwui.text_large_cache_width=4096 \
-    ro.hwui.text_large_cache_height=4096
-- 
2.17.1

